### ISSUE TITLE
Debounce?

### ISSUE THREAD

**Issue by designbyadrian** [souporserious/react-measure#66]
Is it possible to prevent Measure from firing until you stop resizing the window?

I'm currently using react-measure 1.4

**Comment by souporserious** [comment]
If you are using 1.0 you can pass `shouldMeasure={false}` and that will prevent it from updating. Curious if there is any reason you are not using 2.0?

**Comment by designbyadrian** [comment]
But to use `shouldMeasure`, I'd have to listen to window resize events and then trigger Measure. I was hoping to not have it fire while I was resizing the window - only when I release the mouse. Granted, that's how only a few older browser handle it, thus my request for debounce, to have it fire every few hundred milliseconds or so.

I found pre 2.0 easier to use. I've only needed to measure smaller components inside containers, and the wrapping nature of react-measure < 2.0, without the need for a ref, was more straightforward.
